### DISCUSSION TITLE
v0.1.1 now supports reactive declarations

### DISCUSSION THREAD

**Discussion by pintariching** [pintariching/rustle#61] [Announcements]
I've added support for reactive declarations -> `$: double = counter * 2`. I recommend watching [lihautan's video Adding Reactive Declarations into our mini Svelte](https://www.youtube.com/c/lihautan). He explains how they work behind the scenes very well.

For now, not a lot of statements are supported, but more should be relatively easy to add, by expanding `match` statements and extracting the right names. For everything that is not supported, the compiler panics so that it's easier to see what needs to be implemented.

**Comment by qwertydelle** [comment]
Will you be announcing pre release on Reddit anytime soon feel like it will be a good way to attract some contributors.

**Reply by pintariching** [reply]
I did an announcment for the first 0.0.2 pre release version a few days ago. I don't want to spam the subreddit too much. What about posting to This week in Rust?

**Reply by qwertydelle** [reply]
Oh, I didn't know you already posted. You should probably wait a bit and get most of the compiler working first.
